src/09_dictionaries.py

```diff
@@ -34,14 +36,18 @@
 ]
 
 # Add a new waypoint to the list
-# YOUR CODE HERE
+waypoints.append({"lat": -90, "lon": 45, "name": "US?"})
+pp(waypoints)
 
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
 
-# YOUR CODE HERE
+waypoints[0] = {"lat": 43, "lon": -130, "name": "not a real place"}
```

Comment:
Perfectly valid solution! 

What if we specifically, only wanted to modify `lon` and `name`? How can we modify those properties directly with bracket notation?

Reply:
Good call. Thanks for having me research update(). Very good to know!

Reply:
update() is good too! But try this:

`waypoints[0]["lon"]` = -130`

It's similar to how you'd modify a property of an object in swift using dot notation like 

`someObject.property = -130`

except the property name gets wrapped in quotes and brackets: 

`some_object["property"] = -130`